server/config/defaults.js

```diff
@@ -27,6 +27,7 @@ function getDefaults () {
     // core modules
     account: {},
     admin: {},
-    store: {}
+    store: {},
+    plugins: []
```

Comment:
Might be worth checking my latest comment on this one - slight change from our previous discussion, but worth a look.